src/main/java/org/openlmis/requisition/domain/RequisitionTemplate.java

```diff
@@ -136,6 +137,10 @@ public class RequisitionTemplate extends BaseTimestampedEntity {
   @Column(nullable = false)
   private Boolean requisitionReportOnly;
 
+  @Getter
+  @Setter
+  private Boolean patientsTabEnabled;
```

Comment:
let's do it as `boolean`, as I have done here: https://gitlab.com/openlmis/tz-openlmis-commons/-/blob/master/src/main/java/org/openlmis/common/domain/TzRequisitionTemplate.java?ref_type=heads#L48
It will eliminate the confusion from the comment above

Remember to change it for dto also

Reply:
I Changed type to boolean

Reply:
@sradziszewski Did you push the commit with Boolean -> boolean?

Reply:
Yes i did - https://github.com/OpenLMIS/openlmis-requisition/commit/27340a3d51abf2cc9bf6c655ec012ca55b6c3bd2